src/Runner.Common/HostContext.cs

```diff
@@ -282,6 +282,12 @@ public string GetConfigFile(WellKnownConfigFile configFile)
                         ".credentials");
                     break;
 
+                case WellKnownConfigFile.MigratedCredentials:
+                    path = Path.Combine(
+                        GetDirectory(WellKnownDirectory.Root),
+                        ".credentials_migrated");
```

Comment:
only self-hosted runners that currently authed with SPS will get is file created as part of the migration.
once we have all FF turn on and shutdown SPS, any newly configured self-hosted runner will only have `.credentials` file.